docs/semgrep-code/notifications.md

```diff
@@ -56,15 +56,149 @@ To view, add, remove, disable, or enable your saved channels:
+2. In the selected Slack channel, enter the following slash command: `/semgrep_subscribe`.
+3. **Optional**: Enter the name of a specific project after `/semgrep_subscribe` to receive findings for that specific project only, for example, `/semgrep_subscribe acme-corp/vulnerable-repo`. The project must be entered in the following format:
+    <br />
+    <code>/semgrep_subscribe <span className="placeholder">ACCOUNT_NAME/REPOSITORY_NAME</span></code>
+4. Choose an organization in the list under **Select target organization**. The dialog box expands with additional options.
+    ![Semgrep Slack app dialog box for subscribing to notifications](/img/semgrep-slack-subscribe.png "Semgrep Slack app dialog box for subscribing to notifications")
+5. **Optional**: Set up additional filters.
+    1. For Semgrep users that receive both Semgrep Code findings and Semgrep Supply Chain vulnerabilities, you can select **target scan types** to subscribe to either Semgrep Code, Semgrep Supply Chain, or both.
+    2. Select any number of policies to receive findings for under the **Selected Policies** field. By default, you are subscribed to all policies, including the Monitor policy. This can potentially result in a noisy message.
+6. Click **Subscribe**. If you did not specify a project after `/semgrep_subscribe`, the channel is subscribed to findings from all your repositories in Semgrep Cloud Platform.
+7. **Optional**: To set up Slack notifications for additional workspaces, repeat steps 1 to 6. The Semgrep Slack integration is set up on a per-workspace basis.
+
+You have successfully set up notifications for Semgrep findings. The Semgrep Slack app reports new findings after every scan. It does not report findings that were previously discovered.
+
+:::tip Suggested workflow
+In your Slack workspace, create separate channels for either policies, repositories (projects), or types of findings depending on your business or development need. This ensures that developers receive only findings that are relevant to them. 
+:::
+
+![Message from Semgrep Slack app consolidating findings after a scan](/img/slack-findings-message.png#bordered "Message from Semgrep Slack app consolidating findings after a scan")
+_Figure 1_. A sample Slack message with Semgrep findings.
+
+#### Removing notifications for findings in Slack
+
+:::note
+This operation removes or unsubscribes a channel from notifications. To uninstall the Semgrep Slack App, refer to [Uninstalling the Semgrep Slack App from your Slack workspace
+](#uninstalling-the-semgrep-slack-app-from-your-slack-workspace).
+:::
+
+To remove or unsubscribe to notifications:
+
+1. Enter the channel that will unsubscribe from Semgrep findings.
+2. Type `/semgrep_unsubscribe`.
+3. Select the target organization to unsubscribe from.
+4. Click **Unsubscribe**.
+
+You have unsubscribed from Semgrep finding notifications for that particular channel.
+
+#### Making changes to your Slack notifications
```

Comment:
Optional: This header may sound too unspecific. Changes of what kind? Making a suggestion below, feel free to dismiss it.
```suggestion
#### Changing Slack notifications settings
```

Or (this might be even less specific though :D ):
```suggestion
#### Changing your Slack notifications
```